core-web/libs/block-editor/src/lib/nodes/ai-content/ai-content.node.ts

```diff
@@ -0,0 +1,76 @@
+import { Node } from '@tiptap/core';
+
+declare module '@tiptap/core' {
+    interface Commands<ReturnType> {
+        AIContentNode: {
+            insertAINode: (content?: string) => ReturnType;
+        };
+    }
+}
+
+export const AIContentNode = Node.create({
+    name: 'aiContent',
+
+    addAttributes() {
+        return {
+            content: {
+                default: ''
+            }
+        };
+    },
+
+    parseHTML() {
+        return [
+            {
+                tag: 'div'
+            }
+        ];
+    },
```

Comment:
This function gets the HTML DOM element passed as a parameter. The `div` tag can interfere with other extensions, we should consider changing the selector to something like `div[data-ai-node]`